src/meta/app.ts

```diff
@@ -49,7 +49,8 @@ export const ClientRoutes = {
 
   Admin: {
     Root: newInstance<void>('admin'),
-    Section: newInstance<{ sectionName: AssessmentHomeRouteNames }>(':sectionName'),
+    Section: newInstance<{ sectionName: AssessmentHomeRouteNames }>('admin', ':sectionName'),
+    User: newInstance<{ id: number }>('admin', 'users/:id'),
```

Comment:
Why both user and section ? Use only one

Reply:
Also how come this is `'users/:id' instead of `'users', ':id'`

Reply:
this is the subrouter of the admin section:

```
      <Routes>
        {sections.map(({ name, component }) => (
          <Route key={name} path={name} element={React.createElement(component, {})} />
        ))}

        <Route path={ClientRoutes.Admin.User.path.relative} element={<User />} />

        <Route path="*" element={<Navigate to={AdminRouteNames.userManagement} replace />} />
      </Routes>
```

the admin section name is dynamic as the home sections. at the moment we have userManagement, in the future we will add dataExport.

the admin/users/:id is for the user form, as the user form in home.

about 'users/:id'... I already done this for home. the problem is that the router wants a relative path and, at the moment, splitting that address to get the relative path returns only :id:

```
    path: {
      absolute,
      relative: parts[parts.length - 1],
    },
```